src/pages/patch/PatchRestartModal.tsx

```diff
@@ -90,6 +93,7 @@ export const PatchRestartModal: React.FC<PatchModalProps> = ({
 
   const patchAnalytics = usePatchAnalytics();
   const handlePatchRestart = async (e): Promise<void> => {
+    console.log(childPatches);
```

Comment:
```suggestion
```

Reply:
I have to keep it there because otherwise it errors because childPatches is not used. Implementing it to actually use it will take some time.

Reply:
I would have advocated for removing it from the component. It will log undefined for every user who uses the patchRestartModal

Reply:
It's temporary. It'll be removed once EVG-14597 is done.

Reply:
Ah understood! Going forward is this maybe a good use case for `@ts-ignore`?